client/src/pages/Signin.js

```diff
@@ -9,181 +9,219 @@ import logo_pt from "../assets/images/favicon.jpeg";
+                        margin="normal"
+                        name="password"
+                        value={details.password}
+                        onChange={changeHandler}
+                        sx={{ width: "90%" }}
+                        error={errors.password}
+                        helperText={
+                            errors.password && "Please enter you password."
+                        }
+                    />
+                    <div style={{ width: "90%" }}>
+                        {userState?.isErrors === true && (
+                            <p style={{ marginTop: "30px", color: "red" }}>
+                                {userState?.errorMessage?.authForms}
+                            </p>
+                        )}
 
-            <Button
-              sx={{
-                marginTop: 3,
-                width: "100%",
-                height: "50px",
-              }}
-              style={{
-                borderRadius: 3,
-                backgroundColor: "black",
-                padding: "10px 10px",
-                fontSize: "20px",
-              }}
-              variant="contained"
-              type="submit"
+                        <Button
+                            sx={{
+                                marginTop: 3,
+                                width: "100%",
+                                height: "50px",
+                            }}
+                            style={{
+                                borderRadius: 3,
+                                backgroundColor: "black",
+                                padding: "10px 10px",
+                                fontSize: "20px",
+                            }}
+                            variant="contained"
+                            type="submit"
+                        >
+                            Login
+                        </Button>
+                        <Grid container>
+                            <Grid
+                                item
+                                sx={{
+                                    display: "flex",
+                                    width: "100%",
+                                    marginTop: "20px",
+                                    justifyContent: "space-between",
+                                }}
+                            >
+                                <p>Don't have an account ?</p>
+                                <Link to="/sign-up" className="link-redirect">
+                                    Sign Up
+                                </Link>
+                            </Grid>
+                            <li>
+                                <p
+                                    onClick={handleOpenr}
+                                    style={{
+                                        fontSize: "16px",
+                                        color: "blue",
+                                        textDecorationLine: "underline",
```

Comment:
@ChahitKolte2112  just write `cursor:"pointer"` style property just below this line